pkg/utils/config.go

```diff
@@ -0,0 +1,133 @@
+package utils
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	color "github.com/threeal/bro/pkg/utils/color"
+)
+
+type Config interface {
+	Read() error
+	Write() error
+	init()
+}
+
+var HOME_DIR = getHomeDirectory()
+var BRO_CONFIG_DIR = filepath.Join(*HOME_DIR, ".bro")
+
+type BackendConfig struct {
+	ListenAddr *string `json:"listen_addr"`
+}
+
+type ClientConfig struct {
+	BackendAddr *string `json:"backend_addr"`
+}
```

Comment:
Let's put `BackendConfig` in `pkg/bro-backend` and `ClientConfig` in `pkg/bro`. We may need to rename that type since it's clear that those configs are for backend/client.

Reply:
what do you mean by "rename that type"? should i just rename it "Config" and place them in their respective directory?

Reply:
yep, exactly.